Kudu.Services.Web/updateNodeModules.cmd

```diff
@@ -10,7 +10,7 @@ set counter=0
 set /a counter+=1
 echo Attempt %counter% out of %attempts%
 
-cmd /c npm install https://github.com/projectkudu/KuduScript/tarball/16de31b5f5ca590ea085979e5fa5e74bb62f647e
+cmd /c npm install https://github.com/projectkudu/KuduScript/tarball/a37efe5669a6f479d11cffeade3706cfdd67c358
 IF %ERRORLEVEL% NEQ 0 goto error
```

Comment:
has the kuduscript change been merged to projectkudu/kuduscript?

Reply:
not yet, but the download still works

Reply:
understood.   I am afraid if it did not get merged by any reason in kuduscript, we should not merge here either.   That 's why we should review/merge there first.  @ahmelsayed  could help with the review..